app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command, old_file, new_file = command.split()
```

Comment:
It is better to use a different name for a variable `command` that contains only part of an entire `command`